app/src/forms/attestations/forestrysectoroperatorscreening/silvipc-import.js

```diff
@@ -0,0 +1,704 @@
+        const submissionCount = await Models.Submission.query().count().first();
+        const attestationCount = await Models.Attestation.query().count().first();
+        const businessCount = await Models.Business.query().count().first();
+        const contactCount = await Models.Contact.query().count().first();
+        const locationCount = await Models.Location.query().count().first();
+        const statusCount = await Models.Status.query().count().first();
+        const noteCount = await Models.Note.query().count().first();
+
+        console.log(`Submissions: ${submissionCount.count}`);
+        console.log(`Attestations: ${attestationCount.count}`);
+        console.log(`Businesses: ${businessCount.count}`);
+        console.log(`Contacts: ${contactCount.count}`);
+        console.log(`Locations: ${locationCount.count}`);
+        console.log(`Statuses: ${statusCount.count}`);
+        console.log(`Notes: ${noteCount.count}`);
+
+        let form = await Models.Form.query()
+          .first()
+          .throwIfNotFound();
+
+        let version = await Models.Version.query()
+          .first()
+          .where('formId', form.formId)
+          .modify('orderDescending')
+          .throwIfNotFound();
+        form.versions = [version];
+        form.formVersionId = version.formVersionId;
+
+        console.log('import ipc_plans...');
+        let skipped = 0;
+        let trx;
+        try {
+          trx = await transaction.start(ImportSubmission.knex());
+          for (const ipcPlan of importData.ipcPlans) {
+            const exists = await Models.Submission.query(trx).findById(ipcPlan.ipcPlanId);
+            if (exists) {
+              console.log(`IPC Plan ${ipcPlan.ipcPlanId} has already been imported.  Skipping...`);
+              skipped++;
+            } else {
+              // create the submission from ipcPlan
+              const submission = {
+                submissionId: ipcPlan.ipcPlanId,
+                formVersionId: form.formVersionId,
+                confirmationId: ipcPlan.ipcPlanId.substring(0, 8).toUpperCase(),
+                type: ipcPlan.operationType,
+                deleted: false,
+                createdBy: ipcPlan.createdBy,
+                createdAt: moment.utc(ipcPlan.createdAt).toISOString(),
+                updatedBy: ipcPlan.updatedBy,
+                updatedAt: moment.utc(ipcPlan.updatedAt).toISOString()
+              };
+              //
+              // attestation
+              //
+              const attestation = {...ipcPlan};
+              // remove invalid fields/data.
+              delete attestation.ipcPlanId;
+              delete attestation.deletedAt;
+              delete attestation.notes;
+              delete attestation.formVersion;
+              delete attestation.operationType;
+              // update to new model fields
+              attestation.submissionId = submission.submissionId;
+              attestation.attestationId = submission.submissionId;
```

Comment:
Is this used? We don't have an attestationId field do we?

Reply:
nm, wrong model I was looking at

Reply:
yes, ipc_plan basically ended up in 2 tables, submission and attestation.  attestation does have it's own id, but it's never used in the api.